### ISSUE TITLE
Support for NETCOREAPP and NETSTANDARD2.0

### ISSUE THREAD

**Issue by bizzbizz** [lbugnion/mvvmlight#74]
`SimpleIoc` shows up as `SimpleIoc : ISimpleIoc` and breaks everything. While it could be like before : `SimpleIoc : ISimpleIoc, IServiceLocator` with a simple change in the preprocessors.

Is dotnet core going to be supported any time soon?

**Comment by wonea** [comment]
Yes please, I've just started my migrations to NET Core 3.0 and have warnings.

![image](https://user-images.githubusercontent.com/557192/65596324-87e82980-df8e-11e9-94ef-0d1b3f21ca08.png)

**Comment by michael-hawker** [comment]
🦙 For .NET Standard support please try migrating to the MVVM Toolkit: https://docs.microsoft.com/en-us/windows/communitytoolkit/mvvm/migratingfrommvvmlight
